src/libraries/System.Net.Security/src/System/Net/Security/SslApplicationProtocol.cs

```diff
@@ -10,13 +10,16 @@ namespace System.Net.Security
     public readonly struct SslApplicationProtocol : IEquatable<SslApplicationProtocol>
     {
         private static readonly Encoding s_utf8 = Encoding.GetEncoding(Encoding.UTF8.CodePage, EncoderFallback.ExceptionFallback, DecoderFallback.ExceptionFallback);
+        private static readonly byte[] s_http3Utf8 = new byte[] { 0x68, 0x33 }; // "h3"
         private static readonly byte[] s_http2Utf8 = new byte[] { 0x68, 0x32 }; // "h2"
         private static readonly byte[] s_http11Utf8 = new byte[] { 0x68, 0x74, 0x74, 0x70, 0x2f, 0x31, 0x2e, 0x31 }; // "http/1.1"
 
         // Refer to IANA on ApplicationProtocols: https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids
-        // h2
+        /// <summary>Defines a <see cref="SslApplicationProtocol"/> instance for HTTP 3.0.</summary>
+        public static readonly SslApplicationProtocol Http3 = new SslApplicationProtocol(s_http3Utf8, copy: false);
```

Comment:
@karelz, should this require preview?  Should https://github.com/dotnet/runtime/blob/79c4144f174754bf4fa18827a292549214f86581/src/libraries/System.Net.Primitives/src/System/Net/HttpVersion.cs#L17?

Reply:
ATM, we don't do that for `HttpVersion.Http3` since you can easily construct the version manually and this is very similar.

Either way, if we decide to annotate this, we should be consistent and do the same thing with Http3 version.